LogoREGIUI/Sources/Features/OrderEntryFeature/DiscountStackFeature/DiscountStackView.swift

```diff
@@ -54,5 +54,8 @@ struct DiscountStack: View {
             }
             .padding(.top, 20)
         }
+        .onAppear{
```

Comment:
nits: SwiftFormat的にはスペース入れた方がいいかも

必要ならこのファイルごとフォーマッタかけてもいいかも！（ `control + shit + I`）
```suggestion
        .onAppear {
```